apps/ltn/src/partition.rs

```diff
@@ -35,6 +35,9 @@ pub struct Partitioning {
 
     use_expensive_blockfinding: bool,
     pub broken: bool,
+
+    #[serde(skip_serializing, skip_deserializing)]
```

Comment:
When we do save, it'd literally just be the polygon (in lat/lng) and maybe user-defined name